{
	"cve": {
		"CVE_data_meta": {
			"ASSIGNER": "security-advisories@github.com",
			"ID": "CVE-2022-35969"
		},
		"data_format": "MITRE",
		"data_type": "CVE",
		"data_version": "4.0",
		"description": {
			"description_data": [
				{
					"lang": "en",
					"value": "TensorFlow is an open source platform for machine learning. The implementation of `Conv2DBackpropInput` requires `input_sizes` to be 4-dimensional. Otherwise, it gives a `CHECK` failure which can be used to trigger a denial of service attack. We have patched the issue in GitHub commit 50156d547b9a1da0144d7babe665cf690305b33c. The fix will be included in TensorFlow 2.10.0. We will also cherrypick this commit on TensorFlow 2.9.1, TensorFlow 2.8.1, and TensorFlow 2.7.2, as these are also affected and still in supported range. There are no known workarounds for this issue."
				}
			]
		},
		"problemtype": {
			"problemtype_data": [
				{
					"description": [
						{
							"lang": "en",
							"value": "CWE-617"
						}
					]
				}
			]
		},
		"references": {
			"reference_data": [
				{
					"name": "https://github.com/tensorflow/tensorflow/commit/50156d547b9a1da0144d7babe665cf690305b33c",
					"tags": [
						"Patch",
						"Third Party Advisory"
					],
					"url": "https://github.com/tensorflow/tensorflow/commit/50156d547b9a1da0144d7babe665cf690305b33c"
				},
				{
					"name": "https://github.com/tensorflow/tensorflow/commit/50156d547b9a1da0144d7babe665cf690305b33c",
					"tags": [
						"Patch",
						"Third Party Advisory"
					],
					"url": "https://github.com/tensorflow/tensorflow/commit/50156d547b9a1da0144d7babe665cf690305b33c"
				},
				{
					"name": "https://github.com/tensorflow/tensorflow/security/advisories/GHSA-q2c3-jpmc-gfjx",
					"tags": [
						"Patch",
						"Third Party Advisory"
					],
					"url": "https://github.com/tensorflow/tensorflow/security/advisories/GHSA-q2c3-jpmc-gfjx"
				},
				{
					"name": "https://github.com/tensorflow/tensorflow/security/advisories/GHSA-q2c3-jpmc-gfjx",
					"tags": [
						"Patch",
						"Third Party Advisory"
					],
					"url": "https://github.com/tensorflow/tensorflow/security/advisories/GHSA-q2c3-jpmc-gfjx"
				}
			]
		}
	},
	"impact": {
		"baseMetricV3": {
			"cvssV3": {
				"attackComplexity": "LOW",
				"attackVector": "NETWORK",
				"availabilityImpact": "HIGH",
				"baseScore": 7.5,
				"baseSeverity": "HIGH",
				"confidentialityImpact": "NONE",
				"integrityImpact": "NONE",
				"privilegesRequired": "NONE",
				"scope": "UNCHANGED",
				"userInteraction": "NONE",
				"vectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
				"version": "3.1"
			},
			"exploitabilityScore": 3.9,
			"impactScore": 3.6
		}
	},
	"configurations": {
		"CVE_data_version": "4.0",
		"nodes": [
			{
				"cpe_match": [
					{
						"cpe23Uri": "cpe:2.3:a:google:tensorflow:*:*:*:*:*:*:*:*",
						"versionEndExcluding": "2.7.2",
						"versionStartIncluding": "2.7.0",
						"vulnerable": true
					},
					{
						"cpe23Uri": "cpe:2.3:a:google:tensorflow:*:*:*:*:*:*:*:*",
						"versionEndExcluding": "2.8.1",
						"versionStartIncluding": "2.8.0",
						"vulnerable": true
					},
					{
						"cpe23Uri": "cpe:2.3:a:google:tensorflow:2.10:rc1:*:*:*:*:*:*",
						"vulnerable": true
					},
					{
						"cpe23Uri": "cpe:2.3:a:google:tensorflow:2.10:rc2:*:*:*:*:*:*",
						"vulnerable": true
					},
					{
						"cpe23Uri": "cpe:2.3:a:google:tensorflow:2.10:rc3:*:*:*:*:*:*",
						"vulnerable": true
					},
					{
						"cpe23Uri": "cpe:2.3:a:google:tensorflow:2.10:rc0:*:*:*:*:*:*",
						"vulnerable": true
					},
					{
						"cpe23Uri": "cpe:2.3:a:google:tensorflow:*:*:*:*:*:*:*:*",
						"versionEndExcluding": "2.9.1",
						"versionStartIncluding": "2.9.0",
						"vulnerable": true
					}
				],
				"operator": "OR"
			}
		]
	},
	"lastModifiedDate": "2024-11-21T07:12Z",
	"publishedDate": "2022-09-16T21:15Z"
}
